modules/table/src/test/java/org/apache/ignite/internal/table/RecordBinaryViewOperationsTest.java

```diff
@@ -410,4 +413,17 @@ void assertEqualsValues(SchemaDescriptor schema, Tuple expected, Tuple actual) {
     @NotNull private TableImpl createTableImpl(SchemaDescriptor schema) {
         return new TableImpl(new DummyInternalTableImpl(), new DummySchemaManagerImpl(schema), null);
     }
+
+    public <T extends Throwable> void assertThrowsWithCause(Class<T> expectedType, Executable executable) {
```

Comment:
maybe it's time to introduce some TestUtils. WDYT?

Reply:
I think we should check a public exception here and error code, once we will introduce codes and public exceptions.
So, an assertThrowsWithCause must be replaced to junit assertThrows with a correct public exception.

We check a cause here, which is an internal exception, as a temporal solution unless we have public classes and error codes.